### DISCUSSION TITLE
Starstruck achievement not showing

### DISCUSSION THREAD

**Discussion by nirajgiriXD** [community/community#62574] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have already got more than 16 stars in this [repo](https://github.com/nirajgiriXD/github-score/) but also the starstruck achievement is not showing in my profile.

**Comment by hammadalive7** [comment]
IT take some time. 
For more details you can visit: https://github.com/Schweinepriester/github-profile-achievements

**Comment by smaaliyan** [comment]
The "Starstruck" achievement on GitHub is awarded when you've received 25 stars on a repository. Since you mentioned you have more than 16 stars but haven't received the achievement yet, you'll need to get 25 stars on that repository to unlock the "Starstruck" achievement on your profile.

**Reply by arithefirst** [reply]
This is false.
![image](https://github.com/user-attachments/assets/fdda09b7-afeb-4ec1-850b-e919cbd59dd4)

**Comment by radar-blip** [comment, marked as answer]
@nirajgiriXD, you have the starstruck achievement badge, it is just hidden from your profile.

Go here: https://github.com/nirajgiriXD?tab=achievements click on the badge, then click on the eye icon to unhide it.

**Reply by nirajgiriXD** [reply]
@linuxlawson Thanks

**Reply by radar-blip** [reply]
Anytime. 👍🏻
